Magento_Cms/layout/cms_noroute_index.xml

```diff
@@ -0,0 +1,14 @@
+<page layout="1column" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
```

Comment:
this handle is a good solution, 'cause it could be perhaps used for /no-route and /asdsa slug of 404 page. but your HW asked you to create a new page layout and apply that from the admin panel. follow todos please